content/memory_model.tex

```diff
@@ -37,7 +37,7 @@
 \openshmem dynamic memory allocation routines (\textit{shpalloc} and
 \textit{shmem\_malloc}) allow collective allocation of \emph{Symmetric Data
 Objects} on a special memory region called the \emph{Symmetric Heap}. The
-Symmetric Heap is created during the execution of a program at a memory location
+Symmetric Heap is created, during the execution of a program, at a memory location
```

Comment:
This seems like a misapplication of _Comma Use No. 3_ in [Purdue's Extended Rules for Using Commas](https://owl.english.purdue.edu/owl/resource/607/02/).